wsgiservice/resource.py

```diff
@@ -586,6 +649,18 @@ def get_request_data(self):
         """
         request_data = [self.path_params, self.request.GET]
 
+        # Workaround for BEEKEEPER Android Apps
```

Comment:
This work around should be removed. We fixed it some time ago. May be consult with the mobile team real quick.